### ISSUE TITLE
Add two new options in the package

### ISSUE THREAD

**Issue by tuxkernel** [cspiel/typog#6]
Inverted exclamation mark (`¡`) and inverted question mark (`¿`) match very well in lowercases. But in all caps must be raised to the baseline (Harmony, Robert Bringhurst dixit).

![all-caps](https://github.com/user-attachments/assets/e21c3f11-c4cb-45f1-9bc9-a683de70338b)

This example is _only_ applied in spanish (castilian, asturian, etc.), portuguese, italian, and so on. We need add two new options:

- `raisecapitalquestion=<dimen>`, and
- `raisecapitalexclamation=<dimen>`.

and macros:

- `\capitalquestion`, and
- `\capitalexclamation`.

Minion Pro was well crafted by Robert Slimbach:

![minionpro-example](https://github.com/user-attachments/assets/e6b60639-cb65-4b8b-bf53-ffcf532993a7)

The example is useful in headings and body text with all caps. It's just another suggestion. Best wishes ;-)

**Comment by cspiel** [comment]
I confirm the need for y-adjusted inverted exclamation marks and
inverted question marks looking at your examples and explanation.

However, your option and macro names are misleading (at least to
a person who does not speak Spanish) as they imply manipulations
of the question mark or exclamation mark.  If I have understood
correctly, you are interested in shifting the upside-down versions
not the regular ones.

Moreover, I doubt whether two separate adjustment dimensions
are required.  Both, inverted question mark and inverted
exclamation mark are designed so similarly that a single vertical
shift possibly fixes both.  Show me three examples of fonts
where different values are required and I'll agree with your
suggestion.  FF TheSans seems to be a good start.  :wink:

**Comment by tuxkernel** [comment]
Yes, _only_ upside-down versions (`¿`, `¡`) not the regular ones (`?`, `!`). English speakers _only_ use one (?) at the end of sentences, but spanish speakers we use two; one in the beginning (`¿`, `¡`) and one at the ending of sentences (`?`, `!`). Example: 

- ¿Cómo estás, Chris? (How are you, Chris?)
- ¡Bien! ¿Y tú? (Fine! and you?)
- ¡Bien también! (Fine too!)
- Fue un gusto verte, Chris. (It was nice to see you, Chris)
- A mí también, Tuxkernel (Me too, Tuxkernel).

The dialogue was typeset mostly in lowercases (match very well with ascenders and descenders), but in all caps they need to be elevated to the baseline. I never I've used an american keyboard layout, but in my spanish keyboard layout, I'm using `SHIFT + ¡` to produce inverted exclamation mark; and `SHIFT + ?` to produce inverted question mark. I repeat, those marks _only_ are useful in all caps (spanish, portuguese, italian, and so on).

Now, related to option names, they were named just to illustrate the problem (suggested). The names of new options could be:

- `raisecapitalinvertedquestionmark=<dimen>`. Probably not a value or dimen, just aligned to the baseline.
- `raisecapitalinvertedexclamationmark=<dimen>`. Probably not a value or dimen, just aligned to the baseline.

Maybe we don't even need new options in the package, just macros for it:

-`\capitalinvertedquestionmark`, and
-`\capitalinvertedexclamationmark`.

Babel package (spanish) includes two macros for this: `"!` (¡) and `"?` (¿).  Look:

![babel](https://github.com/user-attachments/assets/a3773111-ff2c-49f5-b591-2f04b13b3b8a)

These macros are useful for TeX spanish users, but not for a english speaker typesetting paragraphs in different languages. TeX hackers can easily do the trick (rotate those marks) using graphicx (\rotatebox command) or rotating packages. I repeat, perhaps we don't need new options in the package, just simple macros. I was clear?
